web/lib/rpi-auth.js

```diff
@@ -42,11 +43,18 @@ function getLogoutRedirectUri() {
 
 function getRegisterRedirectUri() {
   const params = new URLSearchParams();
-  params.set('brand', 'coderdojo');
+  params.set('brand', brand);
```

Comment:
I might well be missing something, but where is `brand` being set?

Reply:
It's just a string const being set up on line 18

Reply:
Ha, of course - I should really look at the whole file rather than the diff :)